frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.unit.spec.tsx

```diff
@@ -109,17 +112,26 @@ describe("ExpressionWidget", () => {
       // enter in expression editor should not trigger "onChangeExpression" as popover is not valid with empty "name"
       expect(onChangeExpression).toHaveBeenCalledTimes(0);
 
+      // The name must not be empty and must not consist of spaces only.
```

Comment:
> The name must not be empty

1. Assertion for empty name is missing.

> must not consist of spaces only.

2. The logic isn't only about spaces but about all whitespace characters. Let's assert some tab characters as well and update the comment.